src/collections/_documentation/development/sdk-dev/event-payloads/exception.md

```diff
@@ -15,7 +21,14 @@ except Exception as e:
     raise ValueError() from e
 ```
 
-The order of exceptions would be `Exception` and then `ValueError`.
+The order of exceptions reported would be `Exception` and then `ValueError`.
```

Comment:
_nit_: What does "reported" here signal? They are both reported at the same time in one event.

Reply:
Assuming you mean `/store` endpoint. How about:

```
The order of exceptions in the list would be `Exception`, then `ValueError`.
```

We could also make it a code block:

```
The order of exceptions in the list would be:

```json
[
  {"type": "Exception" },
  {"type": "ValueError" }
]
``
```

Reply:
> Assuming you mean `/store` endpoint.

Yes, that's what I meant, sorry. We call it the "reporting API"

```
For information about the reporting API see SDK Development.
```
https://docs.sentry.io/api/

---

I'll use the prose form, since we already have code examples later on which are more complete.

I find it confusing to have out-of-context JSON blobs in documentation, because it is not obvious where exactly it is supposed to go, and then easy to get the nesting wrong (just like I did with the `mechanism` example in this PR).